src/views/Products/ProductEdit.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-container class="my-12" fluid>
+    <v-card class="mx-auto pa-10" width="800" height="auto">
+      <template v-slot:title>
+        <div class="mx-auto text-center" style="font-size: 24px">
+          {{ pageTitle }}
+        </div>
+      </template>
+
+      <v-sheet width="300" class="mx-auto">
+        <v-form
+          ref="form"
+          @submit.prevent="handleSubmit"
+          @keydown.enter.prevent
+        >
+          <ProductCreateAndEditForm
+            :productInfo="productInfo"
+            :clearTable="clearTable"
+          />
+
+          <form-buttons @reset-form="resetForm" />
+        </v-form>
+      </v-sheet>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup>
+import {
+  prepareProductsContent,
+  prepareResourcesContent,
+} from "@/utils/data-formatter";
+import { ref, computed, inject, onMounted } from "vue";
+import { useRoute, useRouter } from "vue-router";
+import { useStore } from "vuex";
+import ProductCreateAndEditForm from "@/components/Form/ProductCreateAndEditForm.vue";
+
+const props = defineProps(["VDataTable"]);
+const store = useStore();
+const route = useRoute();
+const router = useRouter();
+const productId = route.params.productId;
+const snackbarProvider = inject("snackbarProvider");
+const clearTable = ref(false);
+
+const user = computed(() => store.getters["auth/getUser"]).value;
+const productInfo = ref({});
+const pageTitle = ref(route.meta.title);
+const form = ref(null);
+
+onMounted(async () => {
+  await fetchResourcesForUser();
+});
+
+if (pageTitle.value.includes("Edit")) {
```

Comment:
We shouldn't rely on the title. Please, use the route!

Reply:
fixed